app/team.go

```diff
@@ -1049,6 +1049,14 @@ func (a *App) GetTeamMembersByIds(teamID string, userIDs []string, restrictions
 	return teamMembers, nil
 }
 
+func (a *App) GetCommonTeamIDsForUsers(userIDs []string) ([]string, *model.AppError) {
```

Comment:
The naming here can be a little confusing. 
It meant to me for multiple users until I digged further and saw it's actually only meant between two users.
What about renaming it to something like `GetCommonTeamIDsForTwoUsers` or `GetCommonTeamIDsBetween2Users` or `GetCommonTeamIDsForDirectMessages`. Something that more explicitly mentions its for two users rather than having to dig into the the store method to see that if `len() != 2` then it throws error.

Reply:
Good idea.